event-based-simulators/main/scripts/README.md

```diff
@@ -0,0 +1,32 @@
+# Generic scripts information
+This scripts folder contains:
+- The **ExportOrImportProfileData.py** which is used to export measurements or alarms data from devices
+- The **Environment.py** which is used to setup environment parameters
+
+## Setup environment parameters:
+Input base url, tenant ID, username, password in Environment.py file
+
+## Install cumulocity-python-api package
+Follow the instructions in: https://github.com/SoftwareAG/cumulocity-python-api
+
+### Installation from PyPI
+```shell
+pip install c8y_api
+```
+
+## Run the script
+If the environment parameters were not setup, they can be input as arguments when running the script
+```shell
+python ExportProfilesData.py [-h] [--baseurl BASEURL] [--id-tenant ID_TENANT]
```

Comment:
- using the correct file name works & shows the options & how to use it
- however, after showing the options, a few lines of stacktrace is shown which makes it hard to read
- can the stacktrace be removed from the output?
`Traceback (most recent call last):`